README.md

```diff
@@ -352,22 +360,27 @@ const oktaAuth = new OktaAuth({
 });
 
 export default App = () => {
-  const history = useHistory();
+  const navigate = useNavigate();
 
   const customAuthHandler = (oktaAuth) => {
     // Redirect to the /login page that has a CustomLoginComponent
     // This example is specific to React-Router
-    history.push('/login');
+    navigate('/login');
```

Comment:
no change here. `navigate` should only be a replacement of `restoreOriginalUri` from `oktaAuth` to decouple `react-router` dependency in `<Security>`.

Reply:
react-router v6 has no `useHistory` hook, but added `useNavigate ` instead
https://github.com/ReactTraining/react-router/blob/dev/docs/advanced-guides/migrating-5-to-6.md#use-navigate-instead-of-history